assets/debugbar.css

```diff
@@ -157,6 +157,7 @@ a.phpdebugbar-tab.phpdebugbar-active span.phpdebugbar-badge {
   a.phpdebugbar-tab span.phpdebugbar-badge {
     display: none;
     margin-left: 5px;
+    float: right;
```

Comment:
I don't think a PR should address 2 issues at the same time. I've copied this minor change to my `Use Laravel Mix` PR.

A PR should cover a single issue, not multiple.